### DISCUSSION TITLE
I can't use only IPV6 on IP Pools

### DISCUSSION THREAD

**Discussion by AbdelBTS** [postalserver/postal#1992] [Help with using Postal]
Hi guys,

I want to use only IPV6 on IP Pools, but when i try to save it. A error message appear and say "IPV4 can't be blank".

This is possible to disable IPV4 on IP Pools ?

Thanks

**Comment by willpower232** [comment]
I don't think that is currently possible I'm afraid, you might be able to enter some garbage IPv4 like 999.999.999.999 but I'm not sure how Postal would handle that.

**Comment by AbdelBTS** [comment]
I tried this method but it did not work.  Does this mean that postal cannot send email via IPV6?  In this case why is an IPv6 box available in the IP Pools page?

**Reply by willpower232** [reply]
Postal expects you to operate both addresses so if your setup only has IPv6 then I'm not sure what to suggest

**Reply by AbdelBTS** [reply]
Is there no way to disable this obligation in settings via SSH?
I only use IPv6 without that my whole project is canceled :/

**Reply by AbdelBTS** [reply]
I really do not understand why, because IPV6 must replace Ipv4, what is the point of forcing the use of Ipv4.  Can you report my problem to the developer.  You will save my life thank you very much in advance
